Sources/StreamChatUI/ChatMessageList/MessageComposer/ChatMessageConfirmEditButton_Tests.swift

```diff
@@ -0,0 +1,85 @@
+//
+// Copyright © 2021 Stream.io Inc. All rights reserved.
+//
+
+import StreamChat
+import StreamChatTestTools
+@testable import StreamChatUI
+import SwiftUI
+import XCTest
+
+class ChatMessageConfirmEditButton_Tests: XCTestCase {
+    private lazy var container = UIView().withoutAutoresizingMaskConstraints
+
+    override func setUp() {
+        super.setUp()
+        container.subviews.forEach { $0.removeFromSuperview() }
+    }
+
+    func test_defaultAppearance_newMessage() {
+        let view = ChatMessageConfirmEditButton().withoutAutoresizingMaskConstraints
+        container.embed(view)
+
+        view.isEnabled = true
+        AssertSnapshot(container, variants: .onlyUserInterfaceStyles, suffix: "new-enabled")
+
+        view.isEnabled = false
+        AssertSnapshot(container, variants: .onlyUserInterfaceStyles, suffix: "new-disabled")
+    }
+
+    func test_appearanceCustomization_usingUIConfig() {
+        var config = UIConfig()
+        config.images.messageComposerSendEditedMessage = UIImage(
+            named: "reaction_love_big",
+            in: .streamChatUI
+        )!
+            .tinted(with: .systemPink)!
```

Comment:
Same here, would be better if these images would be present inside the folder `TestImages` inside `StreamChatUITests`